scripts/apps/monitoring/directives/MonitoringGroup.js

```diff
@@ -412,6 +412,10 @@ export function MonitoringGroup(cards, api, authoringWorkspace, $timeout, superd
                 }
 
                 return apiquery(criteria, true).then((items) => {
+                    if (config.features.autorefresh) {
```

Comment:
I think we should give it some more specific name, in settings it's hard to guess what is autorefreshed

Reply:
autorefreshLists?

Reply:
yep, or monitoringAutorefresh, but it's not only in monitoring right?

Reply:
is it in search too

Reply:
maybe autorefreshContent then?